src/main/java/me/fnfal113/sfchunkinfo/commands/ScanChunk.java

```diff
@@ -55,7 +59,7 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
     }
 
     public void getAmount(Chunk chunk, Player player){
-        for(int y = 0; y <= 319; y++) {
+        for(int y = 0; y <= chunk.getWorld().getMaxHeight(); y++) {

```

Comment:
y values can be negative also.

Reply:
you can use this one https://github.com/Slimefun/Slimefun4/blob/1f3d0f5c20bf0220457c80475ad5cf62c4b91e01/src/main/java/io/github/thebusybiscuit/slimefun4/utils/WorldUtils.java

Reply:
from that to this WorldUtils.getMinHeight(chunk.getWorld())?